spec/rubocop/cop/performance/array_semi_infinite_range_slice_spec.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Performance::ArraySemiInfiniteRangeSlice, :config do
+  subject(:cop) { described_class.new(config) }
+
+  # TODO: The following is no longer required when RuboCop 0.78 or lower support will be dropped.
+  # https://github.com/rubocop-hq/rubocop/pull/7605
+  let(:ruby_version) { 2.7 }
+
```

Comment:
Can you add tests for `TargetRubyVersion <= 2.6`?

Reply:
Endless ranges were introduced in 2.7, so in <= 2.6 they produce an invalid syntax error. Should I test for this?

Reply:
Ah, Right! My bad! 💦